apps/src/lib/kits/maker/BoardController.js

```diff
@@ -159,9 +160,9 @@ export default class BoardController {
   }
 
   onBoardEvent(component, event, callback) {
-    // TODO (bbuchanan): Add accelerometer events for "singletap" and "doubletap" that map to
-    // subsets of the "tap" event
-    // https://docs.google.com/document/d/1VuDefx4wijBkyiap-36qpfCAoNEzu5B7yLldhhhuv7U/edit#bookmark=id.daat8jt8eda8
+    if (BOARD_EVENT_ALIASES[event]) {
+      event = BOARD_EVENT_ALIASES[event];
+    }
```

Comment:
Love it when previously-established structure makes it possible for additions like this to be clean and simple!